### PR TITLE
Use schema from the api

### PR DESCRIPTION
**Pull request by zregvart**
We do not want to maintain two copies of schema generation logic. This
builds on PR#286[1] to use the schema generated in the controller/api,
instead of generating the schema at runtime.

[1] enterprise-contract/enterprise-contract-controller#286

### CODE REVIEW

**Comment by zregvart** [comment]
This needs enterprise-contract/enterprise-contract-controller#286 to be merged and the version to be bumped here before merging.

**Review by lcarva — APPROVED** [review]
LGTM! (Once the ecp reference is properly updated)

It would be great to add some kind of test to ensure the issue this is fixing is actually fixed. (Or maybe that should go in the controller repo?) Not a blocker for this PR.

**Review by robnester-rh — APPROVED** [review]
LGTM!

**Comment by zregvart** [comment]
This is no longer using the version from my fork, merging...

**Pull request merged**
